### PR TITLE
platformLayers Demo app use cases

### PR DESCRIPTION
**Pull request by dima-avdeev-jb**
Collect some use cases for testing platfromLayers

image

image

### CODE REVIEW

File: compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/scene/UIViewComposeSceneLayer.uikit.kt [hunk 1]
```diff
@@ -149,7 +149,7 @@ internal class UIViewComposeSceneLayer(
         onPreviewKeyEvent: ((KeyEvent) -> Boolean)?,
         onKeyEvent: ((KeyEvent) -> Boolean)?
     ) {
-        //todo
+        //todo It needs to handle dismiss key, like Esc. But on iOS it is very rare case.
```
Comment (MatkovIvan) [inline compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/scene/UIViewComposeSceneLayer.uikit.kt:152]: It's [exposed](https://github.com/JetBrains/compose-multiplatform-core/blob/bd10a452e07119ba036b3b56987dc8747bdb3d44/compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/window/Popup.skiko.kt#L346-L347) to public, so it's not about only dismiss key.

Comment (dima-avdeev-jb) [reply compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/scene/UIViewComposeSceneLayer.uikit.kt:152]: Thanks, TODO comment updated

File: compose/mpp/demo/src/commonMain/kotlin/androidx/compose/mpp/demo/components/layers/PlatformLayers.kt [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+package androidx.compose.mpp.demo.components.layers
+
+import androidx.compose.mpp.demo.Screen
+
+/**
+ * Test cases for platform layers
+ */
+val PlatformLayers = Screen.Selection(
+    "PlatformLayers",
```
Comment (MatkovIvan) [inline compose/mpp/demo/src/commonMain/kotlin/androidx/compose/mpp/demo/components/layers/PlatformLayers.kt]: It's not about "PlatformLayers", it's common code. I'd suggest just adding a new button to existing `DialogExample` page

Comment (dima-avdeev-jb) [reply compose/mpp/demo/src/commonMain/kotlin/androidx/compose/mpp/demo/components/layers/PlatformLayers.kt]: I want to collect cases to test platformLayers=true/false and compare it with Desktop.

Comment (dima-avdeev-jb) [reply compose/mpp/demo/src/commonMain/kotlin/androidx/compose/mpp/demo/components/layers/PlatformLayers.kt]: Maybe we can move it to another page after stabilizing platformLayers?

Comment (dima-avdeev-jb) [reply compose/mpp/demo/src/commonMain/kotlin/androidx/compose/mpp/demo/components/layers/PlatformLayers.kt]: Decided to place these examples on 2 pages

**Review by MatkovIvan — APPROVED** [review]

**Pull request merged**